Lumina/Modules/BuildMonitor/View/BuildView.swift

```diff
@@ -55,6 +62,12 @@ struct BuildView: View {
             }
         }
     }
+    
+    private func copyBuildNumber() {
+        let pasteboard = NSPasteboard.general
+        pasteboard.clearContents()
```

Comment:
Seems to be the recommendation to clearContents before setting a string to the pasteboard.